### PR TITLE
Wp 4

### PR DESCRIPTION
**Pull request by OfryL**

### CODE REVIEW

File: locomotion-sdk/src/Components/RideHistoryTable/index.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import propsTypes from 'prop-types';
+import i18n from '../../I18n';
+import { TableContainer, StyledHeaderRow, StyledRows, textStyle } from './styled';
+
+const rowsFlexWidth = [2, 3, 3];
+
+const RideHistoryTable = ({ data }) => {
+  const getCreatedAt = date => {
+    const Date = moment(date).format('DD-MM-YYYY');
+    const Time = moment(date).format('HH:MM');
```
Comment (oriefrati) [inline locomotion-sdk/src/Components/RideHistoryTable/index.js]: start variable name in lower case

File: locomotion-sdk/src/services/one-signal.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+};
+
+class NotificationsService {
+  constructor() {
+    this.network = network;
+  }
+
+  init = () => {
+    console.log('INIITTTT');
```
Comment (OfryL) [inline locomotion-sdk/src/services/one-signal.js]: rm

**Review by OfryL — COMMENTED** [review]
👍

File: locomotion-sdk/src/Components/RideHistoryTable/styled.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { StyleSheet, ScrollView, View } from 'react-native';
+import styled from 'styled-components';
+import { Row, Rows } from 'react-native-table-component';
+
+const tableMarginFromSides = '20';
+
+export const TableContainer = styled.ScrollView(`
```
Comment (oriefrati) [inline locomotion-sdk/src/Components/RideHistoryTable/styled.js]: you can remove the brackets

File: locomotion-sdk/src/pages/RideHistory/index.js [hunk 4]
```diff
@@ -1,16 +1,49 @@
...
+  <NoRidesMessageContainer>
+    <NoRidesTitle>
+      <NoRidesTitleText>{i18n.t('rideHistory.noRides')}</NoRidesTitleText>
+      <NoRidesTitleSubText>{i18n.t('rideHistory.noRidesSubText')}</NoRidesTitleSubText>
+    </NoRidesTitle>
+  </NoRidesMessageContainer>
+);
 
 export default ({ navigation }) => {
+  const [rides, setRides] = useState({});
```
Comment (oriefrati) [inline locomotion-sdk/src/pages/RideHistory/index.js]: change the default to null

File: locomotion-sdk/src/services/ride-history.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import network from './network';
+
+const RideHitory = {
+  getHistory: async () => {
+    const req = await network.get('api/v1/me/rides/history');
+    const { data: res } = req;
+    return res || {};
```
Comment (oriefrati) [inline locomotion-sdk/src/services/ride-history.js]: you can do it in easier way:
`
const { data } = await network.get('api/v1/me/rides/history');
return data;
`

**Review by oriefrati — APPROVED** [review]

**Review by oriefrati — APPROVED** [review]

**Pull request merged**